### PR TITLE
Remove spire server info requirement from the frontend #214

### PR DESCRIPTION
**Pull request by mamy-CS**
Addresses: https://github.com/spiffe/tornjak/issues/214
- PR includes some error handling and user notification

### CODE REVIEW

File: tornjak-frontend/src/redux/actions/types.ts [hunk 1]
```diff
@@ -140,10 +143,14 @@ export interface SpireHealthCheckingAction extends Action<typeof GLOBAL_SPIRE_HE
     payload: boolean;
 }
 
-export interface SpireHealthCheckTimeAction extends Action<typeof GLOBAL_SPIRE_HEALTH_CHECK_Time> {
+export interface SpireHealthCheckTimeAction extends Action<typeof GLOBAL_SPIRE_HEALTH_CHECK_TIME> {
     payload: SpireHealtCheckFreq;
```
Comment (maia-iyer) [inline tornjak-frontend/src/redux/actions/types.ts:147]: nit, doesn't have to be this PR, but maybe we should fix this typo wherever it is relevant?

File: tornjak-frontend/src/components/tornjak-server-info.tsx [hunk 2]
```diff
@@ -40,29 +46,68 @@ type TornjakServerInfoProp = {
...
+          <div>
+            <InlineNotification
+              kind="warning"
+              hideCloseButton
+              lowContrast
+              title="Note: No Server Config Provided from server. Tornjak Backend does not have access to SPIRE config!"
+            />
+          </div>
+        )
+      }
```
Comment (maia-iyer) [inline tornjak-frontend/src/components/tornjak-server-info.tsx:110]: I like the look of these warnings!

**Review by maia-iyer — APPROVED** [review]
LGTM! Tested end-to-end. Just one important nit comment about the type of debugserver

File: tornjak-frontend/src/redux/actions/types.ts (hunk 1 above)
Comment (mamy-CS) [reply tornjak-frontend/src/redux/actions/types.ts:147]: resolved

File: tornjak-frontend/src/components/types.ts [hunk 3]
```diff
@@ -81,6 +81,28 @@ export interface EntriesList {
...
+                trust_domain: string
+            },
+            expires_at: number,
+            subject: string
+        }
+    ],
+    uptime: number,
+    federated_bundles_count: number
+}
+
```
[the remark this answers is not in the document]
Comment (mamy-CS) [reply tornjak-frontend/src/components/types.ts:105]: Yes good catch, added a note!

**Pull request merged**